### ISSUE TITLE
Error creating new block using @wordpress/block

### ISSUE THREAD

**Issue by JessPinkman** [WordPress/gutenberg#22285]
**Describe the bug**
Cannot npm init

terminal throws:
npx: installed 86 in 4.921s
/home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/semver/semver.js:323
    throw new TypeError('Invalid Version: ' + version)
    ^

TypeError: Invalid Version: 0.32
    at new SemVer (/home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/semver/semver.js:323:11)
    at SemVer (/home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/semver/semver.js:313:12)
    at /home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/check-node-version/index.js:118:33
    at module.exports (/home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/map-values/index.js:9:18)
    at /home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/check-node-version/index.js:112:22
    at end (/home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/run-parallel/index.js:18:15)
    at done (/home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/run-parallel/index.js:22:10)
    at each (/home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/run-parallel/index.js:28:7)
    at /home/user/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/run-parallel/index.js:38:43
    at /home/jesspinkman/.npm/_npx/7918/lib/node_modules/@wordpress/create-block/node_modules/check-node-version/index.js:59:14

**To reproduce**
1.. type npm init @wordpress/block test
2. error

**Editor version (please complete the following information):**
- WordPress version: 5.4.0
- Default installation

**Desktop (please complete the following information):**
 - OS: Ubuntu (via WSL2 windows 10)
- Semver: 5.4.1
- npm: 6.14.5
- node: 12.16.2

**notes**
deprecated npm init wordpress-block does work

**Comment by talldan** [comment]
@JessPinkman Is that from following the instructions here?
https://github.com/WordPress/gutenberg/tree/master/packages/create-block

It worked for me, though I'm on a macbook, so perhaps it's an OS specific issue. I'll add the `Needs Testing` label.

I'll also change the title to make this a little easier to find.

**Comment by JessPinkman** [comment]
> @JessPinkman Is that from following the instructions here?
Yes problem occurs right away when using bash command npm init @wordpress/block test-folder

**Comment by talldan** [comment]
I'm not sure really. Looking at the error, the issue seems to stem from an incorrect version number `0.32` being used somewhere. Semantic version numbers should be made up of three parts e.g. `0.32.0` or `0.0.32`.

Not sure if that rings any bells. I'll try to test this on my Windows partition at some point, but would also welcome any other Windows users to test this.

**Comment by gziolo** [comment]
I tested both `es5` and `esnext` templates on macOS and it works correctly.

**Comment by dkress59** [comment]
I am having almost the same issue on Ubuntu 20.04, although my invalid version says `git+0.32`:

```
$ sudo npx @wordpress/create-block
npx: Installierte 217 in 7.887s
/root/.npm/_npx/6781/lib/node_modules/@wordpress/create-block/node_modules/semver/semver.js:323
    throw new TypeError('Invalid Version: ' + version)
    ^

TypeError: Invalid Version: 0.32+git
```
I have tried cleaning all caches, updating node and reinstalling npm via https://npmjs.org/install.sh without success.

**Comment by dkress59** [comment]
@JessPinkman I solved this for my machine: I had installed yarn without properly configuring the Debian package repository, so Ubuntu installed `cmdtest` as `yarn`. Apparently `@wordpress/create-block` uses yarn to install packages, and since I had the wrong yarn installed, that's why it threw an error and aborted.

The fix: Install yarn as documented [here](https://classic.yarnpkg.com/en/docs/install/#debian-stable).
